app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <resources>
     <string name="app_name">SuperHeroArch</string>
-    <string name="main_button">Click me</string>
+    <string name="main_button">Obtener Personajes</string>
```

Comment:
For this application at the moment no translations will be made and it will only be seen in Spanish, because it is to test other development concepts.